### PR TITLE
Support geoip2

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by okkez** [comment]
TODO:
* Add test using geoip2_compat and geoip2_c
  * On TravisCI, Ubuntu Precise and Trusty does not have libmaxminddb-dev, so we must use Docker on TravisCI. This task needs lots of time.

See also #39

**Comment by okkez** [comment]
@y-ken Any concern? If you're OK, I will add test to this PR.

File: lib/fluent/plugin/geoip.rb [hunk 1]
```diff
@@ -116,35 +129,51 @@ def parse_json(message)
...
       @geoip_lookup_key.each do |field|
-        address.store(field, record[field]); next if not record[field].nil?
-        key = field.split('.')
-        obj = record
-        key.each {|k|
-          break obj = nil if not obj.has_key?(k)
-          obj = obj[k]
-        }
-        address.store(field, obj)
+        address[field] = record[field] || record.dig(*field.split('.'))
```
Comment (repeatedly) [inline lib/fluent/plugin/geoip.rb]: `dig` is introduced since ruby v2.3.0, so hard to use in this plugin.

Comment (okkez) [inline lib/fluent/plugin/geoip.rb]: Right. But I've added dependency to [dig_rb gem](https://github.com/jrochkind/dig_rb) to use `dig` method.

Comment (repeatedly) [inline lib/fluent/plugin/geoip.rb]: Ah, I missed it. Awesome :)

**Comment by y-ken** [comment]
Hi @okkez 
Oops, I have merged this PR.
Would you please add test for geoip2?
But we need to discuss for new version style.
How to contact to you?

**Comment by okkez** [comment]
OK, I will add test for geoip2 in next month as soon as possible.
I will create new PR, please wait.

> But we need to discuss in offline to solve issue quickly.
> How to contact to you?

Could you mail me using my e-mail address in git commit author info?

**Comment by y-ken** [comment]
I got it!

**Comment by repeatedly** [comment]
Do we have a chance to release new version?

**Comment by y-ken** [comment]
@repeatedly Sorry, I have released as v0.7.0.
https://rubygems.org/gems/fluent-plugin-geoip

**Pull request merged**